event_stage_cancelled/data/mail_template_data.xml

```diff
@@ -179,6 +179,6 @@
 </td></tr>
 </table>
         </field>
-        <field name="lang">{{ object.partner_id.lang }}</field>
+        <field name="lang">{{ object.event_id.lang or object.partner_id.lang }}</field>
```

Comment:
Why to choose event lang over partner's?

Reply:
It is to be consistent with what core does in the event email templates.